stdlib/smtplib.pyi

```diff
@@ -149,6 +150,16 @@ class SMTP_SSL(SMTP):
 LMTP_PORT: int
 
 class LMTP(SMTP):
-    def __init__(
-        self, host: str = ..., port: int = ..., local_hostname: str | None = ..., source_address: _SourceAddress | None = ...
-    ) -> None: ...
+    if sys.version_info >= (3, 9):
+        def __init__(
+            self,
+            host: str = ...,
+            port: int = ...,
+            local_hostname: str | None = ...,
+            source_address: _SourceAddress | None = ...,
+            timeout: float = ...,
```

Comment:
This will still cause a stubtest error, because the default has type `object` at runtime. Can you add a comment to stubtest_allowlists/py39.txt and stubtest_allowlists/py310.txt about this?

Reply:
stubtest actually special cases object in default values, because the sentinel pattern is very common: https://github.com/python/mypy/blob/e557ec242b719e386a804766b0db8646f86ef0e3/mypy/stubtest.py#L362